packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -53,6 +55,12 @@ TableColumnTextColumnConfig
     }
 }
 
+/**
+ * The table column for displaying strings.
+ */
+// prettier-ignore
+export class TableColumnText extends fractionalWidthColumn(TableColumnTextBase) {}
```

Comment:
seeing it in use it feels like the mixin name isn't very descriptive, what about one of:

```ts
export class TableColumnText extends applyFractionalWidthColumnMixin(TableColumnTextBase) {}
export class TableColumnText extends mixinFractionalWidthColumn(TableColumnTextBase) {}
export class TableColumnText extends mixinFractionalWidthColumnAPI(TableColumnTextBase) {}
```

for comparison, how it looks to apply the arai mixin:
```ts
applyMixins(Banner, ARIAGlobalStatesAndProperties);
```

I think I lean to `mixinFractionalWidthColumnAPI` but open to whatever you prefer